packages/server/src/api/routes/tests/automation.spec.ts

```diff
@@ -456,14 +456,13 @@ describe("/automations", () => {
 
         let table = await config.createTable()
 
-        let automation = await filterAutomation()
+        let automation = await filterAutomation(config.appId!)
```

Comment:
There's a `config.getAppId()` function that throws an error when it's not set explaining how to set it, prefer using that. I'd eventually like us to get to the point where these properties are private and can't be used outside of `TestConfiguration`.